core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -861,13 +867,17 @@ void core_mmu_get_user_map(struct core_mmu_user_map *map)
 	} else {
 		map->asid = 0;
 	}
+
+	thread_unmask_exceptions(exceptions);
 }
 
 void core_mmu_set_user_map(struct core_mmu_user_map *map)
 {
 	uint64_t ttbr;
-	uint32_t daif = read_daif();
+	uint32_t daif;
+	uint32_t exceptions = thread_mask_exceptions(THREAD_EXCP_ALL);
 
+	daif = read_daif()
```

Comment:
The daif stuff is interrupt masking so you can remove that now that we can use `thread_mask_exceptions()` instead.

Reply:
ok, thanks.